integreat_cms/cms/models/custom_permissions/custom_permission.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class CustomPermissionManager(models.Model):
```

Comment:
This is mostly personal preference I guess, but I'm not really sure about creating a separate model for this - what do you think about adding the permissions to the user model instead?

Reply:
That would also be fine for me 👍

Reply:
Moved to the user model 👍